polylogue/local_sync.py

```
@@ -264,14 +264,16 @@ def _process_single(session_path: Path) -> Dict[str, object]:
+                target_name = f"{path.name}.{int(time.time())}"
+                trash_path = trash_dir / target_name
+                path.rename(trash_path)
+                import shutil
 
-                    shutil.rmtree(path)
-                else:
-                    path.unlink()
+                shutil.rmtree(trash_path)
                 pruned += 1
             except OSError:
                 continue
```

Ruff (0.15.9)

[warning] `try`-`except` within a loop incurs performance overhead

(PERF203)